tests/backup/backup_helper.go

```diff
@@ -122,6 +124,8 @@ const (
 	rancherActiveCluster                      = "local"
 	rancherProjectDescription                 = "new project"
 	multiAppNfsPodDeploymentNamespace         = "kube-system"
+	storkPodRestartRetryTimeinMinutes         = 15
+	podDestroyTimeout                         = 5 * time.Minute
```

Comment:
y?

Reply:
This was added earlier as part of some testing, we changed the logic, removed this now